recipes/array-record/meta.yaml

```diff
@@ -0,0 +1,74 @@
+{% set name = "array-record" %}
+{% set version = "0.8.0a1" %}
+{% set sha256 = "60d73cd0f038de7a1e8c376b143a5a148b85b1fc316569f4c73a33b8be3dbf80" %}
+{% set bazel_version = "7.2.1" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://github.com/google/array_record/archive/v{{ version }}.tar.gz
+  sha256: {{ sha256 }}
+
+build:
+  number: 0
+  skip: true  # [py<310]
+  skip: true  # [win]
+  skip: true  # [osx]
```

Comment:
```suggestion
```
Let's enable macOS and see what it takes for it to build.  Given that this is a `bazel` build, I won't ask for us to try Windows until the feedstock exists.

Reply:
Okay, `bazel` is a such a painful system to build with we can get by for now only Linux but let's make sure check into macOS arm64 afterward.  The build failure you linked seems to be more of an issue with how `highway-hash` was built than something specific to macOS x86, but it's so difficult to debug bazel that I don't have any suggestions yet.